.github/workflows/spack.yml

```
@@ -33,4 +33,4 @@ jobs:
           spack develop --no-clone --path $GITHUB_WORKSPACE trilinos@develop
           spack add trilinos@develop
           spack concretize -f
-          spack install --cdash-upload-url=https://sems-cdash-son.sandia.gov/cdash/submit.php?project=Trilinos -j16
+          spack install --cdash-upload-url=https://sems-cdash-son.sandia.gov/cdash/submit.php?project=Trilinos --cdash-track='Pull Request' --cdash-build='PR-${{ github.event.pull_request.number }}' -j16
```

```suggestion
          spack install --cdash-upload-url=https://sems-cdash-son.sandia.gov/cdash/submit.php?project=Trilinos --cdash-track='Pull Request' --cdash-build='PR-${{ github.event.pull_request.number }}_spack-trilinos@develop' -j16
```
Adding spack into the build name may be helpful. Could just add `_spack` or `_spack-trilinos@develop`

Currently they're just `PR-{number} - trilinos/kokkos/kokkos-kernels`:
- https://sems-cdash-son.sandia.gov/cdash/index.php?project=Trilinos&filtercount=3&showfilters=1&filtercombine=and&field1=groupname&compare1=61&value1=Pull%20Request&field2=buildname&compare2=63&value2=PR-13808&field3=buildstarttime&compare3=84&value3=NOW

---

Let's just add 'spack', because the spec is still part of the name (e.g. `trilinos@develop...` or `kokkos@develop...`.  Good idea though, I'll add it